### PR TITLE
Update docs: Deploy bookinfo

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>
I updated the pics and some descriptions, please take a look, thanks.
/assign @Patrick-LuoYu @Felixnoo 
#1810

### CODE REVIEW

File: content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ weight: 2400
   使用 `project-admin` 帐户登录控制台并访问您的项目。导航到**项目设置**下的**高级设置**，点击**编辑**，然后选择**编辑网关**。在出现的对话框中，打开**应用治理**旁边的切换开关。
```
Comment (Felixnoo) [inline content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md:23]: I think we need to update `导航到` to a better word? How about `前往`?

File: content/en/docs/quick-start/deploy-bookinfo-to-k8s.md [hunk 2]
```diff
@@ -20,63 +20,60 @@ To provide consistent user experiences of managing microservices, KubeSphere int
 
 - You need to enable **Application Governance**. To do so, perform the following steps:
 
-  Log in to the console as `project-admin` and go to your project. Navigate to **Advanced Settings** under **Project Settings**, click **Edit**, and select **Edit Gateway**. In the dialog that appears, flip on the toggle switch next to **Application Governance**.
+  Log in to the console as `project-admin` and go to your project. Go to **Advanced Settings** under **Project Settings**, click **Edit**, and select **Edit Gateway**. In the displayed dialog box, turn on the toggle switch next to **Application Governance**.
```
Comment (Felixnoo) [inline content/en/docs/quick-start/deploy-bookinfo-to-k8s.md:23]: The description here does not match the current UI. Please check it out.
Xnip2021-07-20_13-45-55

Comment (serenashe) [reply content/en/docs/quick-start/deploy-bookinfo-to-k8s.md:23]: If a user has set a gateway before, then the layout of the page would match the description. I think we need to have a little discussion here.

Comment (serenashe) [reply content/en/docs/quick-start/deploy-bookinfo-to-k8s.md]: I updated the description here by adding a link to another document that has already described this operation. Please take a look, thanks!

File: content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md (hunk 1 above)
Comment (serenashe) [reply content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md]: Changed.

File: content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md [hunk 3]
```diff
@@ -43,40 +41,37 @@ Bookinfo 应用由以下四个独立的微服务组成，其中 **reviews** 微
 
 ### 步骤 1：部署 Bookinfo
 
-1. 使用帐户 `project-regular` 登录控制台并访问项目 (`demo-project`)。导航到**应用负载**下的**应用**，点击右侧的**部署示例应用**。
+1. 使用帐户 `project-regular` 登录控制台并访问项目 (`demo-project`)。前往到**应用负载**下的**应用**，点击右侧的**部署示例应用**。
```
Comment (Felixnoo) [inline content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md:44]: I think `到` should be removed?

File: content/en/docs/quick-start/deploy-bookinfo-to-k8s.md (hunk 2 above)
Comment (Felixnoo) [reply content/en/docs/quick-start/deploy-bookinfo-to-k8s.md]: Cool.

File: content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md (hunk 3 above)
Comment (serenashe) [reply content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md]: Changed.

**Pull request merged**